### PR TITLE
To set mtu on private network

### PR DESCRIPTION
**Pull request by aishwaryabk**
Signed-off-by: Aishwarya Kamat <aishwarya.kamat@ibm.com>

### CODE REVIEW

File: modules/5_install/install.tf [hunk 1]
```diff
@@ -74,7 +79,39 @@ locals {
     }
 }
 
+resource "null_resource" "pre_install" {
+  count      = local.bastion_count
+
+  triggers = {
```
Comment (yussufsh) [inline modules/5_install/install.tf]: Is this trigger needed? Why?

**Review by yussufsh — CHANGES_REQUESTED** [review]
Could you also add it in var.tfvars?

File: modules/5_install/install.tf (hunk 1 above)
Comment (aishwaryabk) [reply modules/5_install/install.tf]: Yes, it could be done without triggers. Thanks

**Comment by aishwaryabk** [comment]
> Could you also add it in var.tfvars?

@yussufsh I have added "private_network_mtu" to the var.tfvars file.

**Comment by yussufsh** [comment]
@aishwaryabk Please help with the conflicts.

**Comment by aishwaryabk** [comment]
> @aishwaryabk Please help with the conflicts.

@yussufsh The conflicts are resolved.

**Review by yussufsh — APPROVED** [review]
/lgtm

**Review by cs-zhang — APPROVED** [review]
/lgtm

**Pull request merged**